plugins/serializers/nowmetric/nowmetric.go

```diff
@@ -0,0 +1,152 @@
+package nowmetric
```

Comment:
Should this parser be called "servicenow"?  When I google for nowmetric I don't get any relevant hits, can you explain a little more?

Reply:
Hi, 
Asked myself the same question...
But ServiceNow usually means "Incident Management" integration which is a complete different story.
nowmetric is the shortest i found, that is branding compliant. (snow is not allowed)